Week3/MAKEME.md

```diff
@@ -112,7 +113,7 @@ let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
         console.log("yes")
     } else {
         console.log("no")
-    }
+    };
```

Comment:
There are missing colons, but not where you added them 🙃

Reply:
ha yes, you are right